src/CXFX.ts

```diff
@@ -0,0 +1,114 @@
+import { Selector } from "./Selector";
+import { genericCleanup, shorten, toFixed } from "./util";
+
+export class CXFX {
+    private tag = "pb-cx";
```

Comment:
All files should be indented with 2 spaces, not 4.

Reply:
should be fixed